tests/lang-elixir.yaml

```diff
@@ -0,0 +1,6 @@
+- desc: it should run elixir
+  command: [elixir,--version]
+  assert:
+  - status == 0
+  - stdout.indexOf("Erlang/OTP 24") != -1 ||
```

Comment:
this our test logic for the version. defaults to this when unversioned, we can look at specific versions in the repo and using a chunk.yaml if needed